foreman_rh_cloud.gemspec

```
@@ -18,6 +18,8 @@ Gem::Specification.new do |s|
   s.test_files = Dir['test/**/*']
   s.test_files -= Dir['test/**/*.orig']
 
+  s.required_ruby_version = '>= 2.7'
```

I like an upper bound
```suggestion
  s.required_ruby_version = '>= 2.7', '< 4'
```